### ISSUE TITLE
Cannot use inherited properties in subclasses?

### ISSUE THREAD

**Issue by Houfeng** [microsoft/node-api-dotnet#365]
This is a great project that has been very useful to me. I noticed a small issue while using it.
  
Cannot use inherited properties in subclasses?
  
Example
```csharp
[JSExport]
public class A {
  public string Name { 
     get {
       //...
     } 
     set {
       //...
     } 
  }
}

[JSExport]
public class B : A {
}
```

Accessing the Name property in JavaScript has no effect，The final generated typescript type definition includes the 'Name' property, but the `.g.cs` generated during compilation did not include the 'Name' property for B.

**Comment by Houfeng** [comment]
In JS, I tried using the following code, which works
```js
Object.setPrototypeOf(B.prototype, A.prototype);
```

**Comment by jasongin** [comment]
Thanks for the bug report. I'll try to fix this soon.

**Comment by Houfeng** [comment]
```js
Object.setPrototypeOf(B.prototype, A.prototype);
````
As mentioned above, accessing parent class properties can work, but there are still issues when calling parent class methods.

```
TypeError: Illegal invocation
```

**Comment by jasongin** [comment]
@Houfeng The fix is available in version `0.8.5`.
